jax_distribution/types.py

```diff
@@ -13,3 +18,20 @@ class PPOTransition(NamedTuple):
     log_prob: chex.Array
     obs: chex.Array
     info: Dict
+
+
+class RunnerState(NamedTuple):
+    """State of the `Runner`."""
+
+    params: FrozenDict
+    opt_state: OptState
+    key: chex.PRNGKey
+    env_state: LogEnvState
+    timestep: TimeStep
+
+
+class Output(NamedTuple):
```

Comment:
I think being slightly more descriptive and calling this `ExperimentOutput` is a good idea.

Reply:
Thanks for all the review points :raised_hands: I will update the name